src/domain/Tables/Table/services/helper.ts

```diff
@@ -66,15 +66,6 @@ const getSelectAllTableCheckboxInputValue = (currentSelectedRows: boolean[]) =>
   return newSelectedAll ? TableCheckboxInputValue.True : TableCheckboxInputValue.False
 }
 
-const handleSortButtonClicked = (name: string, hasSortEnabled: boolean, sort?: IColumnSorts, onSortChange?: (sort: IColumnSorts) => void) => () => {
-  if (sort && onSortChange && hasSortEnabled) {
-    onSortChange({
-      ...sort,
-      [name]: sort[name] === ColumnSortDirection.Descending ? ColumnSortDirection.Ascending : ColumnSortDirection.Descending
-    })
-  }
-}
-
```

Comment:
lul what, was this just superfluous?

Reply:
i beleive so, reading the code it seems so, it would technically be slightly more performant then clicking the title since it doesn't check if it is sorting by it already but that would be negligible